msetnxex.lua

```diff
@@ -0,0 +1,11 @@
+local setExCount = 0
```

Comment:
Nit: here and for all other scripts. I'd maybe group/embed them from smth like `./scripts/XXX.lua`.

Matter of taste, doesn't prevent us from merging.